### PR TITLE
News for JC_139 Polarization and entanglement in baryon–antibaryon pair production in electron–positron annihilation

### PR DESCRIPTION
**Pull request by amitraahul**

### CODE REVIEW

File: _posts/2020-03-27-JC139.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+---
+title:  "Journal Club 139"
```
Comment (xshi) [inline _posts/2020-03-27-JC139.md:2]: Change to more informative one: JC139 - XXXXXXXXX

File: _posts/2020-03-27-JC139.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+date: 2020-03-27 15:00:00 +0800
+author: amitraahul
+categories: jc
+---
+
+<https://indico.ihep.ac.cn//event/11570/>
```
Comment (xshi) [inline _posts/2020-03-27-JC139.md:8]: Release this link with the original paper's link. Since no one can assess our internal indico page.

**Comment by xshi** [comment]
Please fix the above to places.

File: _posts/2020-03-27-JC139.md (hunk 2 above)
Comment (amitraahul) [reply _posts/2020-03-27-JC139.md:8]: https://www.nature.com/articles/s41567-019-0494-8

File: _posts/2020-03-27-JC139.md (hunk 1 above)
Comment (amitraahul) [reply _posts/2020-03-27-JC139.md:2]: JC139_(Polarization and entanglement in baryon–antibaryon pair production in electron–positron annihilation)

File: _posts/2020-03-27-JC139.md [hunk 3]
```diff
@@ -0,0 +1,26 @@
+---
+title:  "Journal Club 139- Nature Paper about Polarization and entanglement in baryon–antibaryon pair production in electron–positron annihilation"
```
Comment (xshi) [inline _posts/2020-03-27-JC139.md:2]: Could you just change to : 

JC139 - Polarization and Entanglement in Baryon-Antibaryon Pair Production in Electron-Positron Annihilation

Comment (xshi) [reply _posts/2020-03-27-JC139.md:2]: This will be easer to be displayed on our website. Thanks!

Comment (amitraahul) [reply _posts/2020-03-27-JC139.md]: Okay. It's done.

**Comment by amitraahul** [comment]
There were some mismatched files which I have removed. Now it is ready to be merged.

**Pull request merged**